### ISSUE TITLE
ipv6-only website check broken

### ISSUE THREAD

**Issue by motp** [keybase/keybase-issues#514]
Keybase fails to prove a website, if said website has _only_ an AAAA record:
## Example: 

host penguinfriendly.de is IPv6-only:
penguinfriendly.de has IPv6 address 2001:7b8:2ff:1a3::2

... and accessible:
wget http://penguinfriendly.de
--2014-04-05 18:54:26--  http://penguinfriendly.de/
Auflösen des Hostnamen »penguinfriendly.de (penguinfriendly.de)«... 2001:7b8:2ff:1a3::2
Verbindungsaufbau zu penguinfriendly.de (penguinfriendly.de)|2001:7b8:2ff:1a3::2|:80... verbunden.
HTTP-Anforderung gesendet, warte auf Antwort... 200 OK
Länge: 4673 (4,6K) [text/html]
[...]
2014-04-05 18:54:26 (66,1 KB/s) - »»index.html«« gespeichert [4673/4673]

... but keybase thinks otherwise:

┌─[bbu@hobbes]─[~]
└──╼ keybase prove web penguinfriendly.de
error: Failed to find a connection to penguinfriendly.de (error #229)

**Comment by motp** [comment]
Any news on this issue?

I just got a mail from your system, stating:

" Hi, motp, your attempted web identity proof of http://penguinfriendly.de never succeeded. We've been checking it repeatedly, and from our perspective, it's either (a) not there, or (b) not posted precisely.

If you think there's a bug, please post an issue on github. Include the link to your proof."

Well, it is still the same issue. The site is up, but keybase doesn't recognize ipv6-only sites.

**Comment by maxtaco** [comment]
You're totally right, we haven't fixed this yet.  I'm very sorry.  We hope to get to it soon.  Thanks for your patience.

**Comment by motp** [comment]
While you are at it ... internationalized domains are also broken in keybase.
I just checked it with https://www.møtp.de

**Comment by motp** [comment]
Well, I can't wait for you guys to wake up. 
I removed the IPv4 DNS entries for my main domain yesterday and since this issue is still unfixed, the proof broke. I assume, you will not fix this for quite some time, so this is the part where I am leaving. 

Have fun. I am out, keybase is just not up to the task.

**Comment by mlindgren80** [comment]
IPv6-only proofs still appear to fail.  Any updates on this?

**Comment by motp** [comment]
Still broken.

**Comment by songgao** [comment]
@maxtaco This is probably simpler now that EC2 supports IPv6

**Comment by songgao** [comment]
We've made progress on this in our staging environment but it's a high risk deployment for us for now. Uptime is an important to us, so we are pushing the prod deploy a little bit. Stay tuned! Sorry for taking so long.

**Comment by andreaso** [comment]
Any progress/ETA on deploying IPv6 in prod?

**Comment by 42wim** [comment]
Will 2018 be the year of IPv6 on the keybase ? :)

**Comment by glguy** [comment]
2019 checking in!

**Comment by 42wim** [comment]
2020 checking in! ;-)

**Comment by lothos612** [comment]
2023 checking in, still no ipv6 lol
